coral/src/app/pages/topics/index.test.tsx

```diff
@@ -58,6 +62,47 @@ describe("Topics", () => {
       expect(headline).toBeVisible();
     });
 
+    it("renders 'Request A New Topic' primary action in heading", async () => {
+      const headline = screen.getByRole("button", {
```

Comment:
Ah, super smol find: the const is probably a `button` not a `headline` :D :D :D 

(I do that all the time, that's why I'm   :D :D :D )

And I would use `.toBeEnabled()` in that case, that makes sure the button is not only visible but also clickable!